README.md

```diff
@@ -187,7 +187,23 @@ let transactionHash = "9FC7D277C1C8ED9CE133CC17AEA9978E71FC644CE6F5F0C8E26F1C635
 let transactionStatus = xrpClient.paymentStatus(for: transactionHash) // TransactionStatus.succeeded
 ```
 
-**Note:** The example transactionHash may lead to a "Transaction not found." error because the TestNet is regularly reset, or the accessed node may only maintain one month of history.  Recent transaction hashes can be found in the [XRP Ledger Explorer ](https://livenet.xrpl.org/).
+**Note:** The example `transactionHash` may lead to a "Transaction not found." error because the TestNet is regularly reset, or the accessed node may only maintain one month of history.  Recent transaction hashes can be found in the [XRP Ledger Explorer ](https://livenet.xrpl.org/).
+
+#### Retrieve specific payment
+
+An `XRPClient` can retrieve a specific payment transaction by hash.
+
+```
```

Comment:
You can write this line as \`\`\`swift and it'll get swift syntax highlighting when github renders it.